src/main/java/teammates/common/datatransfer/questions/FeedbackRankOptionsQuestionDetails.java

```diff
@@ -293,11 +294,12 @@ public String getQuestionResultStatisticsCsv(
 
             List<Integer> ranksAssigned = entry.getValue();
             double average = computeAverage(ranksAssigned);
-            String fragment = option + "," + df.format(average) + Const.EOL;
+            String fragment = option + "," + df.format(average) + ","
+                    + StringHelper.join(",", ranksAssigned) + Const.EOL;
             fragments.append(fragment);
         }
 
-        return "Option, Average Rank" + Const.EOL + fragments.toString() + Const.EOL;
+        return "Option, Average Rank, Ranks Received" + Const.EOL + fragments.toString() + Const.EOL;
```

Comment:
I think it would be more appropriate to show `Ranks received` before `Average rank`. By doing that, the CSV results would also look consistent with the HTML results.